aderyn_driver/src/process_auto.rs

```diff
@@ -71,22 +77,33 @@ pub fn with_project_root_at(
     for (version, file_paths) in &compilation_groups {
         let solc = Solc::find_or_install_svm_version(version).unwrap();
         let solc_bin = solc.solc.to_str().unwrap();
+        println!("{}", &solc_bin);
 
         let command = Command::new(solc_bin)
             .args(remappings.clone())
+            .arg("--base-path")
+            .arg(root_path)
```

Comment:
```suggestion
            .arg(absolute_root_path_str)
```

I needed to change this for it to find the paths correctly.